.eslintrc.js

```diff
@@ -14,8 +14,11 @@ module.exports = {
 
     'globals': {
         '__SMART_BUTTONS__': true,
-        'paypal': true
+        'paypal': true,
+        'jest':     true
+    },
+    "env": {
+        jest: true
     },
```

Comment:
`jest` gets added as a global automatically when you set the env to `jest`. We can also conditionally apply the `jest` env to just the test files.

```suggestion
        'paypal': true
    },
    'overrides': [{
        'files': ['**/*.test.js'],
        'env': { 'jest': true }
    }],
```